lib/users/api.js

```diff
@@ -258,7 +258,7 @@ class API {
 
     let body = await requests.post(
       uri, this.header_, {json: true, body: bodyData});
-    return JSON.parse(body).data;
```

Comment:
All the other `requests.post` instances in this file use `JSON.parse` to parse the return value. This one should not be any different. What's going on?

Reply:
My guess is the `json: true` option also assumes that the response is `application/json`? I really don't know, but it was throwing an error prior to this change. I believe the test for that function also verifies it has the expected fields, too.

Reply:
weird. thanks for catching. please leave a comment explaining this before merging